core/src/test/scala/unit/kafka/log/LogValidatorTest.scala

```diff
@@ -687,6 +687,57 @@ class LogValidatorTest {
     verifyRecordValidationStats(validatedResults.recordValidationStats, numConvertedRecords = 0, records,
       compressed = true)
   }
+  @Test
```

Comment:
Could you please use `ValueSource` or `CsvSource`?

Reply:
Hi @chia7712 thanks for review.
I found valueSource and CsvSource is not an easy way.
How about MehodSource ?

Reply:
```scala
  @ParameterizedTest
  @CsvSource(Array("0,gzip", "1,gzip", "0,lz4", "1,lz4", "0,snappy", "1,snappy"))
  def testInvalidChecksum(code: Byte, compression: String): Unit = {
    checkInvalidChecksum(code, CompressionType.forName(compression))
  }
```

how about this?

Reply:
WoW, this is so great. Thanks!